demo/simulation/custom_panther/compose.yaml

```
@@ -10,13 +10,16 @@ services:
       - /tmp/.X11-unix:/tmp/.X11-unix:rw
       - ./panther.rviz:/ros_ws/src/panther_description/rviz/panther.rviz
       - ./panther.urdf.xacro:/ros_ws/src/panther_description/urdf/panther.urdf.xacro
-    command: roslaunch panther_gazebo panther_simulation.launch
+    command: roslaunch panther_gazebo panther_simulation.launch use_gpu:=true
```

```suggestion
    command: roslaunch panther_gazebo panther_simulation.launch
```
Container lacks GPU access